package.json

```diff
@@ -28,6 +28,7 @@
     "react-native-calendars": "^1.17.4",
     "react-native-config": "^0.11.5",
     "react-native-easy-markdown": "^1.1.9",
+    "react-native-linear-gradient": "^2.4.0",
```

Comment:
Using the same shadow as in #101. Can't use native shadows here, because Android shadows are never on top of a component.